src/Raven.Studio/typescript/widgets/virtualGrid/columns/textColumn.ts

```diff
@@ -91,10 +91,19 @@ class textColumn<T> implements virtualColumn {
         const cellValue = this.getCellValue(item);
 
         if (_.isString(cellValue)) {
-            const rawText = this.opts.useRawValue && this.opts.useRawValue(item) ? cellValue : generalUtils.escapeHtml(cellValue);
+            const extraClasses = this.opts.extraClass ? this.opts.extraClass(item) : "";
```

Comment:
let's avoid having this customization on textColumn - this is generic class and we don't want to introduce any such customizations here. Either create subclass of textcolumn or handle that outside.

Reply:
done - handled outside